src/vtbackend/ViInputHandler.cpp

```diff
@@ -385,7 +386,7 @@ Handled ViInputHandler::sendKeyPressEvent(Key key, Modifiers modifiers, Keyboard
     if (modifiers.any())
         return Handled { true };
 
-    auto const keyMappings = std::array<std::pair<Key, std::string_view>, 10> { {
+    auto const keyMappings = std::vector<std::pair<Key, std::string_view>> { {
```

Comment:
Why did you go for vector here? It is a minor, but we could avoid the unnecessary associations when sticking with array.

Reply:
I switched to vector to make it a bit easier to update this list in the future, so you can just add it with one line, without the need to update number of elements, if you think that it is better to keep array here i can undo it